src/avfoundation.cs

```diff
@@ -9160,6 +9160,19 @@ interface AVCapturePhotoSettings : NSCopying
 		[NoWatch, NoTV, NoMac, iOS (12, 0)]
 		[Export ("embedsPortraitEffectsMatteInPhoto")]
 		bool EmbedsPortraitEffectsMatteInPhoto { get; set; }
+
+		[Internal]
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[Export ("availableRawEmbeddedThumbnailPhotoCodecTypes")]
+		NSString[] _GetAvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[Wrap ("Array.ConvertAll (_GetAvailableRawEmbeddedThumbnailPhotoCodecTypes, s => AVVideoCodecTypeExtensions.GetValue (s))")]
+		AVVideoCodecType[] GetAvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[NullAllowed, Export ("rawEmbeddedThumbnailPhotoFormat", ArgumentSemantic.Copy)]
+		NSDictionary RawEmbeddedThumbnailPhotoFormat { get; set; }
```

Comment:
looks like an opportunity for a better (strong typed) return type

Reply:
even if we did not provide it for the non-`Raw` API previously :|

Reply:
@spouliot that is precisely the reason I did not add it. I can do it, no problem.

Reply:
not sure about this one - do we know the format ? might have been the reason it was not done last year

Reply:
I'll take a look and comment here.